packages/renderer/src/lib/task-manager/TaskManagerTabs.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+import { Button } from '@podman-desktop/ui-svelte';
+
+import { IS_TASK_STATUSES } from '/@/stores/tasks';
+import { TASK_STATUSES } from '/@api/taskInfo';
+
+let {
+  searchTerm = $bindable(),
+  onUpdate = (): void => {},
+}: { searchTerm: string; onUpdate?: (searchTerm: string) => void } = $props();
```

Comment:
suggestion (not-blocking): props interface above ?

Reply:
fixed